oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,138 @@
+
+  * **Course Learning Content**, a course-team created package of learning material, with its weekly and sub-weekly structure
+    (i.e., Sections, Subsections, Units, etc) and learning blocks (i.e., leaf xBlocks), tied to specific dates of offering.
+     
+    * This is what we call **(Studio) Course Run**, for the time being.
+    * It is bound to the organization that authors the content.
+    * Note: In the future, this concept may be further separated when we support content reuse across "Course Runs".
+      At that time, we may distinguish between:
+
+      * **Reusable Learning Content** (not tied down to specific dates nor structure) and
+      * **Packaged Learning Content** (an "instantiation" of a "Reusable Learning Content" with structure, dates, and 
+        configuration).
+
+  * **Discoverable Course**, a learner-facing representation of a course that entices enrollment with marketing information
+    such as the course's price and credit/degree outcomes.
+
+    * This is what we call **Catalog Course**, for the time being.
+    * It is bound to a specific marketing site.
+
+  * **Learner Consumable Course**, a learner-engaging course, with read-optimized representation of the learning content. 
+
+    * This is what we call **(LMS) Course Run**.
+    * It is bound to a specific learner.
+
+* Given the above distinctions, each `edX subdomain`_ will manage its own concept. Hence, the authoring logic
+  implementation for each concept will also remain separate. This prevents us from needing to maintain these concepts across
+  multiple domains: keep the authoring of Discoverable-Course (**Catalog Course**) within Discovery; keep the authoring of 
+  Learning-Content-Course (**Course Run**) within Content Authoring. For example, concepts of price, marketing-publish-date,
+  marketing sales, discounts, etc do not need to bleed into the Content Authoring service.
+
+* However, from the course author's perspective, we will provide a unified user experience by having the current Studio tool
+  be the portal to authoring both Course Run metadata and Catalog Course metadata. So even though authoring of Catalog Course
+  metadata will live in the Discovery subdomain, there will be a seamless transition between the front ends of Course Run
+  Authoring and Catalog Course Authoring.
+
+  * This will depend on upcoming architectural infrastructure work on single sign on across the edX microservices and a universal
+    front end header and footer.
+
+* The following is Course Run metadata and its editing should be owned by the `Content Authoring subdomain`_:
+
+  * course run dates
+
+    * start
+    * end
+    * enrollment start
+    * enrollment end
+    * certificate availability
+
+  * pacing type
+
+* The following is Catalog Course metadata and its editing should be owned by the `Discovery subdomain`_:
+
+  * course about image
+  * instructors
+  * seat price
+  * seat type
```

Comment:
As mentioned earlier, seats are owned by ecommerce - the ecommerce.edx.org subdomain